crates/astria-conductor/tests/blackbox/soft_and_firm.rs

```diff
@@ -23,21 +20,13 @@ use crate::{
     mount_update_commitment_state,
 };
 
-/// Tests if a single block is executed and the rollup's state updated (first soft, then firm).
-///
-/// The following steps are most important:
-/// 1. a block at rollup number 1, sequencer height 2 is fetched from Sequencer
-/// 2. the block is executed against the rollup
-/// 3. the rollup's soft commitment state is updated to reflect the last execution
-/// 4. block information for rollup number 1, sequencer height 2 is reconstructed from Celestia
-///    height 1
-/// 5. the rollup's firm commitment state is updated (but without executing the block)
+/// Tests if a single block is executed and the rollup's state updated (first firm, then soft is
+/// ignored).
 ///
-/// NOTE: there is a potential race condition in this test in that the information could be first
-/// retrieved from Celestia before Sequencer and executed against the rollup. In that case step 3.
-/// would be skipped (no soft commitment update).
+/// This is to test for the unlikely scenario of receiving a firm block from Celestia before the
+/// soft block from sequencer.
 #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
-async fn simple() {
+async fn simple_firm_only() {
```

Comment:
Can you give this a more meaningful name? For example:
```suggestion
async fn soft_is_ignored_if_firm_arrives_first() {
```